pages/common/databricks.md

```diff
@@ -0,0 +1,14 @@
+# databricks
+
+> CLI for databricks 
+> More information: <https://docs.microsoft.com/en-us/azure/databricks/dev-tools/cli/>.
+
+- set up host (https address of your databricks environment) and token (generated from databrics' settings)
+
+`databricks configure --token`
+
+
+- list workspace content:
```

Comment:
```suggestion
- List the current workspace content:
```

I don't see where workspace is specified so I assume we are talking about the current workspace.